analysed-packages/pyserial/version-3.5/pyserial-3.5-OSS-disclosure.txt

```diff
@@ -0,0 +1,62 @@
+================================================================================
+
+pyserial-3.5
+
+--------------------------------------------------------------------------------
+
+================================================================================
+
+LICENSES 
+
+--------------------------------------------------------------------------------
+
+BSD-3-Clause
```

Comment:
It is correct that the files of the package are licensed under BSD-3-Clause. The only thing which shall be corrected is the text of the BSD-3-Clause. It seems perhaps a little bit picky but you know we need to be as precise as possible. The text of the BSD-3-Clause license contained in the LICENSE.txt file in the root directory is:

Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions are
met:

\* Redistributions of source code must retain the above copyright
notice, this list of conditions and the following disclaimer.

\* Redistributions in binary form must reproduce the above
copyright notice, this list of conditions and the following
disclaimer in the documentation and/or other materials provided
with the distribution.

\* Neither the name of the copyright holder nor the names of its
contributors may be used to endorse or promote products derived
from this software without specific prior written permission.

THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

This license text contains "*" instead of "1.", "2.", "3." like the license text below. Please correct this. Please make sure that the file (/test/handlers/protocol_test.py) which has the license reference "this is distributed under a free software license, see license.txt" instead of "SPDX-License-Identifier: BSD-3-Clause" like all the others is also updated with the correct license text.